sites/docs/src/content/docs/usage/configuration/introduction.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Configuration
+subtitle: How configure nf-core pipelines
+shortTitle: Configuration options
+weight: 1
+parentWeight: 20
+---
+
+## Configure nf-core pipelines
+
+Each nf-core pipeline comes with a set of “sensible defaults”.
+While the defaults are a great place to start, you will certainly want to modify these to fit your own purposes and system requirements.
+
+When a pipeline is launched, Nextflow will look for config files in several locations.
+As each source can contain conflicting settings, the sources are ranked to decide which settings to apply.
+
+nf-core pipelines may utilize any of these configuration files.
+
+Configuration sources are reported below and listed in order of priority:
+
+1. Parameters specified on the command line (`--parameter`)
+2. Parameters that are provided using the `-params-file` option
+3. Config file that are provided using the `-c` option
+4. The config file named `nextflow.config` in the current directory
+5. The config file named `nextflow.config` in the pipeline project directory
+6. The config file `$HOME/.nextflow/config`
+7. Values defined within the pipeline script itself (e.g., `main.nf`)
+
+While some of these files are already included in the nf-core pipeline repository (e.g., the `nextflow.config` file in the nf-core pipeline repository), some are automatically identified on your local system (e.g., the `nextflow.config` in the launch directory), and others are only included if they are specified using run options (e.g., `-params-file`, and `-c`).
+
+If you clone and manually edit an nf-core pipeline then you cannot update to more recent versions of the pipeline without overwriting your changes. You also risk moving away from the canonical pipeline and losing reproducibility.
```

Comment:
I would be stronger about this, say you SHOULD NOT manually edit an nf-core pipeline downloaded by nextflow